copy.go

```diff
@@ -258,6 +259,9 @@ func copyGraph(ctx context.Context, src content.ReadOnlyStorage, dst content.Sto
 func doCopyNode(ctx context.Context, src content.ReadOnlyStorage, dst content.Storage, desc ocispec.Descriptor) error {
 	rc, err := src.Fetch(ctx, desc)
 	if err != nil {
+		if errors.Is(err, errdef.ErrNotFound) && descriptor.IsForeignLayer(desc) {
```

Comment:
Can we do other way around?
```suggestion
			if descriptor.IsForeignLayer(desc) && errors.Is(err, errdef.ErrNotFound) {
```